jekyll/_cci2/configuration-reference.md

```diff
@@ -1,11 +1,13 @@
 ---
 layout: classic-docs2
-title: "Built-in Job Step Types"
-short-title: "Built-in Job Step Types"
-categories: [getting-started]
-order: 40
+title: "Configuration Reference"
```

Comment:
Is this including all of the options for each section _as well as_ the Built-in Step types? "Configuration Reference" feels a little too broad right now.